src/components/structures/BottomLeftMenu.js

```diff
@@ -52,13 +52,13 @@ module.exports = React.createClass({
             <div className="mx_BottomLeftMenu">
                 <div className="mx_BottomLeftMenu_options">
                     <div className="mx_BottomLeftMenu_createRoom" title="Start chat" onClick={ this.onCreateRoomClick }>
-                        <TintableSvg src="img/icons-create-room.svg" width="24" height="24"/>
+                        <TintableSvg src="img/icons-create-room.svg" width="25" height="25"/>
```

Comment:
what is the rationale for increasing these to 25x25?  The original artwork is 24x24, so scaling it up by 4% is going to turn all of the 1px-wide strokes into fuzzyish 1.04px strokes, especially on non-retina displays...